### DISCUSSION TITLE
L.DomUtil.addClass is not a Method

### DISCUSSION THREAD

**Discussion by gregrowles** [tomickigrzegorz/leaflet-examples#77] [Announcements]
No support for L.DomUtil.addClass in current/latest version of Leaflet;

version: Leaflet 1.9.2+main.3b62c7e
error: Uncaught TypeError: L.DomUtil.addClass is not a function
at: o._makeButton (leaflet-geoman.min.js:1:75562)

**Comment by tomickigrzegorz** [comment]
The documentation says otherwise [domutil](https://leafletjs.com/reference.html#domutil)

**Reply by gregrowles** [reply]
maybe it's the bleeding edge version of Leaflet
/* @preserve
 * Leaflet 1.9.2+main.3b62c7e, a JS library for interactive maps. https://leafletjs.com
 * (c) 2010-2023 Volodymyr Agafonkin, (c) 2010-2011 CloudMade
 */

![image](https://github.com/tomickigrzegorz/leaflet-examples/assets/13313420/0e10631d-26b6-4745-8686-2a7282832c2b)

![image](https://github.com/tomickigrzegorz/leaflet-examples/assets/13313420/c72d3ec8-c589-46ea-938f-a5767845e50a)

FYI, "addClass" is not found inside leaflet.js

**Reply by tomickigrzegorz** [reply]
You're bad at finding information ;)
https://unpkg.com/leaflet@1.9.2/dist/leaflet-src.js
You can easily find addClass

**Edit:**
You have an error in L.DomUtiul.addClass(l,t.className);
but you don't have this error in the first line? How is this possible. You use it there too. t.toggleStatus && L.DomUtil.addClass(n, "active"):
If there was no addClass method, the error would appear in this line first.

**Reply by gregrowles** [reply]
Lol, pls allow me to demonstrate...
![image](https://github.com/tomickigrzegorz/leaflet-examples/assets/13313420/ccb8526c-5c7f-4e9b-aa99-cbe7cb24995a)

I'll revert to 1.9.2 stable for now...

**Reply by tomickigrzegorz** [reply]
I'm not interested in your source, only what is shared by leafeltjs - https://github.com/Leaflet/Leaflet/releases/tag/v1.9.2
Download leaflet.zip, unzip it, open leaflet-src.js and there you will find addClass
The library itself has another entry at the top. There is no "main" there.
This is the official library.
```
/* @preserve
  * Leaflet 1.9.2, a JS library for interactive maps. https://leafletjs.com
  * (c) 2010-2022 Vladimir Agafonkin, (c) 2010-2011 CloudMade
  */
```
Okay, end of discussion, I proved that addClass exists in 1.9.2
